src/styles/blocks/catalog.scss

```diff
@@ -17,9 +17,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 300ms;
```

Comment:
Please specify to which properties you want to apply the transition. 
Currently, it applies to all the properties and that's really bad for the performance 
https://developer.mozilla.org/en-US/docs/Web/CSS/transition